Classes/ContentDefender/Hooks/CommandMapHook.php

```diff
@@ -76,24 +76,21 @@ public function processCmdmap_beforeStart(DataHandler $dataHandler): void
                             $container = $this->containerFactory->buildContainer($parent);
                             $cType = $container->getCType();
                             $allowedConfiguration = $this->tcaRegistry->getAllowedConfiguration($cType, $colPos);
-                            $allowed = true;
                             foreach ($allowedConfiguration as $field => $value) {
                                 $allowedValues = GeneralUtility::trimExplode(',', $value);
                                 if (in_array($recordCType, $allowedValues) === false) {
-                                    $allowed = false;
+                                    $msg = $recordCType . ' is not allowed in ' . $cType;
```

Comment:
Shouldn't that be `$recordCType . ' is not allowed in ' . $colPos;` ?

Reply:
changed it to `$recordCType . ' is not allowed in ' . $cType . ' on colPos ' . $colPos;` (but this message do not appear in "general work"